system/system_error_monitor/config/diagnostic_aggregator/system.param.yaml

```diff
@@ -122,6 +122,12 @@
                   contains: [": Network Traffic"]
                   timeout: 3.0
 
+                ip_packet_reassembles_failed:
```

Comment:
@v-nakayama7440-esol Someone might understand a reassembly error is occurring when this status is OK. Would you consider to change this name with `ip_packet_reassembly`.

Reply:
@v-nakayama7440-esol
Sorry, there is already topic named `CRC Error`. No need to fix.